apps/builder/app/builder/features/sidebar-left/panels/marketplace/templates.tsx

```diff
@@ -169,17 +175,33 @@ export const Templates = ({
         >
           {name}
         </Button>
-        <Link
-          underline="none"
-          href={builderUrl({
-            projectId: projectId,
-            origin: location.origin,
-          })}
-          target="_blank"
-          aria-label="Open project in new tab"
+        <Tooltip
+          content={
+            hasAuthToken
+              ? undefined
+              : 'The project does not have a shared link with "View" permission.'
+          }
         >
-          <ExternalLinkIcon />
-        </Link>
+          <Link
+            css={{
+              "&[data-state=disabled], &[data-state=disabled]:visited": {
```

Comment:
Does :visited overwrites color? Afaik it always use explicitly defined one on unvisited selector.

Reply:
moved to Link style